src/lms-ds-loader/edfi_lms_ds_loader/mssql_lms_operations.py

```diff
@@ -60,7 +62,7 @@ def truncate_staging_table(self, table: str) -> None:
         assert table.strip() != "", "Argument `table` cannot be whitespace"
 
         # Note: for postgresql we'll want `TRUNCATE TABLE {staging} RESTART IDENTITY`
-        self._exec(f"truncate table lms.[stg_{table}];")
+        self._exec(f"TRUNCATE TABLE lms.stg_{table};")
```

Comment:
I changed the SQL statements to use upper case. Neither SQL Server nor PostgreSQL "cares" about the case, but I find it easier to read because it is simple to distinguish the SQL language components from the data structure / names.